stmhal/mpconfigport.h

```diff
@@ -234,6 +234,12 @@ static inline mp_uint_t disable_irq(void) {
 #define free gc_free
 #define realloc gc_realloc
 
+//see STM32F4xx_HAL_CONF.h USE_USB_FS & USE_USB_HS
+// The options are USB_DEVICE_MODE -for any USB_FS or USB_HS but only 1,
+//   if both FS and HS can be devices result undefined
+//Future:
+// USE_USBFS_HOST_MODE possibly part of USB_FS OTG phy
+// USE_USBHS_HOST_MODE possibly part of USB_HS OTG phy
```

Comment:
This comment is a bit confusing.  Probably just say something like "at the moment only USB_FS is supported"